assets/stylesheets/layout/_main.scss

```diff
@@ -275,54 +280,58 @@ html.iframed {
 $sidebar-width-max: 272px;
 $sidebar-width-min: 228px;
 
+.wp {
+	display: flex;
+	flex-direction: column;
+	position: fixed;
```

Comment:
`position: absolute` should work just as well. The only requirement is to have this node take up the entire viewport.